### PR TITLE
Support paymentId in cli API

### LINKED ISSUE #1484: [Feature Request] Add identifier to raidencli API
## Description
The demo train rellies on specific identifiers to process the incoming transactions and differentiate between them.
It would be awesome if you could add the possibility to set an identifier in REST-requests to the raiden-cli.

Right now the API accepts the additional entry int he json but ignores it:
```
pi@NanoPi-Duo2:~$ curl -v -H 'Content-Type: application/json' http://localhost:5001/api/v1/payments/0x31aA9D3E2bd38d22CA3Ae9be7aae1D518fe46043/0x0087b32F69DB0b92cA5F268b499017DC5ca6EBFA -d '{ "amount": 2 , "identifier": 6 }'
```
succeeds but the receiver repots a different identifier:
```
oliver@bbpc:~$ curl localhost:5001/api/v1/payments
[{"identifier": "1588928930581", "amount": "2", "log_time": "2020-05-08T09:08:55.264000", "initiator": "0x00D384EF74575E97884215e9f39142228c7ACfa8", "event": "EventPaymentReceivedSuccess", "token_address": "0x31aA9D3E2bd38d22CA3Ae9be7aae1D518fe46043"}]
```

## Acceptance criteria
- Be able to set an identifier with the raiden-cli

## Tasks
- []

### PR DESCRIPTION
**Pull request by andrevmatos**
**Thank you for submitting this pull request :)**

Fixes #1484

**Short description**
Makes CLI transfer use `identifier` body param from POST request.

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1. Make a payment with CLI passing an `identifier` numeric string param to body
2. Check on logs that identifier was used for transfer

### CODE REVIEW

File: raiden-cli/src/index.ts [hunk 1]
```diff
@@ -73,11 +73,19 @@ async function raidenEnsureChannelIsOpen(
   return channel;
 }
 
-async function raidenTransfer(token: string, target: string, value: BigNumberish) {
+async function raidenTransfer(
+  token: string,
+  target: string,
+  opts: { amount: BigNumberish; identifier?: BigNumberish },
```
Comment (nephix) [inline raiden-cli/src/index.ts:79]: Please move `amount` out of `opts`. It doesn't make sense that `amount` is an option and `token` and `target isnt`

Comment (andrevmatos) [reply raiden-cli/src/index.ts:79]: It does, per API: token and target are URL params on Raiden API, while amount and identifier comes as body params. This is perfectly mirrored here. But I don't have hard feelings about it, and will extract.

Comment (andrevmatos) [reply raiden-cli/src/index.ts]: Done

Comment (nephix) [reply raiden-cli/src/index.ts]: > token and target are URL params on Raiden API, while amount and identifier comes as body params

Apples and oranges. One is how the end user uses it, the other is internal implementation

**Pull request merged**